playground/multisearch.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use function Laravel\Prompts\multisearch;
+
+require __DIR__.'/../vendor/autoload.php';
+
+$models = multisearch(
+    label: 'Which users should receive the email?',
+    placeholder: 'Search...',
+    returnKeys: true,
```

Comment:
There seems to be a bug when `returnKeys` is set to `false`:

![image](https://github.com/laravel/prompts/assets/4977161/556e9283-a6b1-4049-a127-6edc5baa8787)

For the `search` prompt, I opted to have it always return keys to simplify things. Let's do the same here, and if there is a good use case for returning values, then I think that should be a separate PR that adds it to the `search` prompt as well, so they are consistent.

Reply:
Also, for other prompts that support returning keys _or_ values (i.e. `select` and `multiselect`), the behaviour is toggled by passing the options as either an associate array or a list. If we do look at this functionality in a separate PR, ideally it would be the same with whatever you return from the `options` callback.